ext/dom/php_dom.stub.php

```diff
@@ -1696,6 +2113,7 @@ public function remove(string ...$tokens): void {}
         public function toggle(string $token, ?bool $force = null): bool {}
         public function replace(string $token, string $newToken): bool {}
         public function supports(string $token): bool {}
+        /** @virtual */
         public string $value;
```

Comment:
why don't the two properties listed before the methods?

Reply:
The DOM spec also defines the class structure, and I just follow the same ordering that the spec uses. They tend to group things based on functionality similarities. Although idk either why value isn't on top here.